theano/tensor/nnet/abstract_conv.py

```diff
@@ -66,7 +66,6 @@ def get_conv_output_shape(image_shape, kernel_shape,
     """
```

Comment:
In the description, I would add a note that the num_group don't influence the shape. So people don't think it was forgotten.